packages/react/.storybook/Container.js

```diff
@@ -25,8 +25,26 @@ function Container({ story, id }) {
     };
   }, []);
 
+  // Does not include LTS message in UIShell elements to avoid overlaying
+  const url = window.location.href.includes('ui-shell');
+
   return (
     <React.StrictMode>
+      {!url && (
+        <div
+          style={{
+            backgroundColor: 'black',
+            width: 'auto',
+            padding: '1rem',
+            margin: '-1rem -1rem 0px -1rem',
+          }}>
+          <p style={{ color: 'white' }}>
+            Carbon v10 is currently in maintenance mode, support is scheduled to
```

Comment:
Would it make more sense in the storybook to be more explicit on what package and version this applies to? I think the meaning of "v10" could easily be lost on folks.
```suggestion
            carbon-components-react@7.x is in maintenance mode, support is scheduled to
```

Reply:
@tay1orjones Yeah that make sense! 
I also added a link to the release schedule, let me know if looks good.

![Screenshot 2023-10-11 at 09 15 57](https://github.com/carbon-design-system/carbon/assets/52183462/7b7d85aa-4232-4376-8635-d06ce82b245d)